zad2/warriors.rb

```diff
@@ -1,12 +1,55 @@
-class Warrior
-  def initialize(name:, level:)
-  end
+class Character
+  attr_accessor :name :level
 
-  def strength
+  def initialize(name, level)
+    @name = name
+    @level = level
   end
 end
 
+class Warrior < Character
+end
+
+class Monster < Character
+end
+
 class BattleArena
   def initialize(first_character, second_character)
+    @first_character = Warrior.new
+    @second_character = Monster.new
+  end
+
+  def dice_throw
+    dice_points = [1..12].sample
```

Comment:
przypisanie do lokalnej zmiennej jest tutaj niepotrzebne - `dice_points` nie jest używana

Reply:
podejrzewam, że chodziło o przechowanie wyniku rzutu w zmiennej instacji `@dice_points`. Jeśli tak to wystarczy dopisać `attr_writer :dice_points` lub napisać bezpośrednio `@dice_points = [1..12].sample` 😃 więcej o readerach, writerach i accessorach możesz doczytać [tutaj](https://github.com/daftcode/ror4beginners2018/issues/124)